### PR TITLE
Changed a few things

### PR DESCRIPTION
**Pull request by Steve0Greatness**
I didn't do that much. I can list all changes right now:
1. changed the readme to not have heading(level 3s) because it looks better.
2. changed Index be an actual home page.
3. gave images a white background
4. switched from `location.href` to `location.search`, it's better for what was going on with the homepage.
5. gave images a minimum width.
6. Added titles

edit: forgot to mention that I also brought the `index.html` out of `quirks mode`, and added indents on the same page too.

### CODE REVIEW

**Comment by Chiroyce1** [comment]
Thanks for the PR. I will review this when I get some time - will take up to 1-2 days.

File: index.html [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
+    <title>Scratch Assets Image Viewer</title>
+    <script defer src="script.js"></script>
 </head>
-<body></body>
-</html>
+
+<body>
+    <h1>Scratch Assets Image Viewer</h1>
+    <p>To view an image, simply paste the URL into the box below.</p>
+    <input type="url" onchange="loadURL(this)">
```
Comment (Chiroyce1) [inline index.html]: Adding an event-listener in the JavaScript is better.

File: script.js [hunk 2]
```diff
@@ -1,15 +1,35 @@
-let hash = location.href.split("#")[1];
+const search = location.search.substring(1); // Gets the location.search, and then removes the first character
 
-if (hash) {
-    let image = document.createElement('img');
+(function() {
+    if (!search) return; // if the search empty, then it stops.
```
Comment (Chiroyce1) [inline script.js]: Maybe do this instead?
```js
if (search) {
    // Rest of the code
}
```
I'm not a huge fan of the anonymous function there..

File: script.js [hunk 3]
```diff
@@ -1,15 +1,35 @@
...
+
+    // image
+    let image = document.createElement("img");
+    image.src = `https://assets.scratch.mit.edu/get_image/../${search}`;
```
Comment (Chiroyce1) [inline script.js]: What's the `/../` for?

Comment (Steve0Greatness) [reply script.js]: the original string was `https://assets.scratch.mit.edu/get_image/.%2E/${hash}`, `%2E` is the url encoding for `.`(period); so I just replaced it with `.` and it worked. Btw, for some reason `https://assets.scratch.mit.edu/get_image/${hash}` doesn't work

File: script.js (hunk 2 above)
Comment (Steve0Greatness) [reply script.js]: Ok, I could do that. But what do you have against anonymous functions?

File: index.html (hunk 1 above)
Comment (Steve0Greatness) [reply index.html]: but then I'd have to give it an id.

**Comment by Steve0Greatness** [comment]
@Chiroyce1 I implimented the requested changes

File: script.js (hunk 2 above)
Comment (Chiroyce1) [reply script.js]: they make code look bad, not all, but most, see:
```js
// good
setInterval(() => {
    console.log('hi')
}, 300)


// bad
setInterval(function(){
    console.log('hi')
}, 300)
```

File: script.js (hunk 3 above)
Comment (Chiroyce1) [reply script.js]: ok thats fine

**Comment by Chiroyce1** [comment]
Thanks, but it doesn't seem to work..

https://user-images.githubusercontent.com/97374054/161222609-a3f4aa7d-3731-4093-a4c6-a8cf19e90f0e.mov

**Comment by Steve0Greatness** [comment]
Should be fixed(also, nice UI(your browser, not the webpage, I suck at designing))

**Comment by Steve0Greatness** [comment]
@Chiroyce1

**Comment by Chiroyce1** [comment]
@Steve0Greatness I made some minor changes - are you ok with it?

**Comment by Steve0Greatness** [comment]
@Chiroyce1, on line 27 you used an arrow function; Arrow functions cannot be used for things like that, as in that case it only returns the window object. So use an arrow function.

Also, on line 2 it would be nice if you added a `var` or something.

Finally, on line 15, you changed the string appended to the end to have no space; this means that if the image is `foo.svg` then the title would be `foo.svgAssets Image Viewer`.

**Comment by Chiroyce1** [comment]
I added some styling in [this PR](https://github.com/Steve0Greatness/image/pull/1), for some reason GitHub asked me to fork it instead of directly committing...

**Comment by Chiroyce1** [comment]
Found a big problem - we need to fix older links to work with this. Shall I do it?

**Comment by Steve0Greatness** [comment]
Merged

**Comment by Steve0Greatness** [comment]
@Chiroyce1 are you merging soon?

**Comment by Chiroyce1** [comment]
Oops forgot about this 😅

**Pull request merged**